osmaxx/excerptexport/models/output_file.py

```diff
@@ -38,6 +36,13 @@ def download_file_name(self):
     def content_type(self):
         return self.export.file_format
 
+    @property
+    def file_extension(self):
+        if bool(self.file):
+            _discarded, file_extension = os.path.splitext(self.file)
```

Comment:
I guess you're using this name, because just `_` would shadow the `_` from
```python
from django.utils.translation import ugettext_lazy as _
```
?